apps/provisioning/pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -235,3 +235,24 @@ type JobResourceSummary struct {
 	// This may not be an exhaustive list and recommend looking at the logs for more info
 	Errors []string `json:"errors,omitempty"`
 }
+
+// HistoricJob is a history entry of Job. It is used to store Jobs that have been processed.
+//
```

Comment:
NOTE: This resource is used as an append only log and should not be used directly by any external consumer.  When there is a more stable integration with loki (an appropriate append only store) this will be removed without notice.